packages/web/src/components/notification/store/sagas.ts

```diff
@@ -663,10 +665,6 @@ export function* markAllNotificationsViewed() {
   const audiusBackendInstance = yield* getContext('audiusBackendInstance')
   yield* call(waitForBackendSetup)
   yield* call(audiusBackendInstance.markAllNotificationAsViewed)
-  if (NATIVE_MOBILE) {
```

Comment:
cool this is going away now! is it safe to do in this intermediate phase?

Reply:
No this shouldn't have been part of this commit, my b